backend/src/apis/IotaConnector_disabled.test.ts

```diff
@@ -0,0 +1,19 @@
+import { sendDataMessage, getMessage, getAllMessages } from '@/apis/IotaConnector'
+
+jest.mock('@/config', () => ({
```

Comment:
This is not imported, so I guess this has no effect at all.

Reply:
Sorry but it has an effect. If I remove it and switch Iota in my .env on the test will fail.